src/views/Home/components/PrizeTracks/PrizeTracks.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react"
+import "./PrizeTracks.scss"
+import { ReactComponent as HealthHacks } from "images/tracks/health_hacks.svg"
+import { ReactComponent as BeginnerHacks } from "images/tracks/beginner_hacks.svg"
+import { ReactComponent as EarthHacks } from "images/tracks/earth_hacks.svg"
+import { ReactComponent as GoldenStateHacks } from "images/tracks/golden_state_hacks.svg"
+import { ReactComponent as JusticeHacks } from "images/tracks/justice_hacks.svg"
+import { ReactComponent as UIUXHacks } from "images/tracks/ui_ux_hacks.svg"
+
+interface CircleProps {
+  image: any
+  hoverText: string
+}
+
+const Tracks = [
```

Comment:
For keys in React, generally we don't want to use indexes. Since this is a hard-coded array it's fine but just for reference in the future. Also here's a quick [article](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318) if you want to check it out explaining it much more in-depth.